spring-cloud-gcp-dependencies/pom.xml

```diff
@@ -16,10 +16,9 @@
 	<packaging>pom</packaging>
 
 	<properties>
-		<gcp-libraries-bom.version>13.4.0</gcp-libraries-bom.version>
-		<cloud-sql-socket-factory.version>1.0.16</cloud-sql-socket-factory.version>
-		<guava.version>29.0-jre</guava.version>
-		<google-api-client.version>1.30.7</google-api-client.version>
+		<gcp-libraries-bom.version>16.2.0</gcp-libraries-bom.version>
```

Comment:
We probably also have to update this in the new repo

Reply:
Will do.